### ISSUE TITLE
link broken on releases page

### ISSUE THREAD

**Issue by sfraczek** [01org/mkl-dnn#183]
### Description
On releases [releases](https://github.com/01org/mkl-dnn/releases) page link to post-ops api is broken.
I would like to learn more about it. 
Specifically, can I create a fused layer performing conv+relu+pooling?

**Comment by vpirogov** [comment]
Thanks for catching this, I updated [the link](http://01org.github.io/mkl-dnn/structmkldnn_1_1post__ops.html) in the release notes. Currently post-ops API supports fusing ReLU and sum with convolution and ReLU with batch normalization. So you cannot do conv+relu+pooling. See usage example in [Caffe's convolution layer implementation](https://github.com/intel/caffe/blob/f6d01efbe93f70726ea3796a4b89c612365a6341/src/caffe/layers/mkldnn_convolution_layer.cpp).

**Comment by sfraczek** [comment]
Thanks.
Could you also please tell me if BathNorm + ReLU saves memory **and improves speed**? It's hard for me to figure it out.

**Comment by emfomenk** [comment]
BatchNorm + Relu performs: ` dst <-- ReLU(Bnorm(src))`, discarding intermediate result of Bnorm.
Hence less memory read/write happens hence better performance.

**Comment by sfraczek** [comment]
Thanks again 👍 .
